lib/puppet/provider/reboot/windows.rb

```diff
@@ -143,6 +144,19 @@ def package_installer_syswow64?
     end
   end
 
+  def pending_computer_rename?
+    path = 'SYSTEM\CurrentControlSet\Control\ComputerName'
+    active_name = reg_value(path, 'ActiveComputerName')
+    pending_name = reg_value(path, 'ComputerName')
```

Comment:
Does not appear so. I think we may want to add some error handling to the method itself.